xml/templates/message_templates/contribution_offline_receipt_html.tpl

```diff
@@ -21,7 +21,10 @@
   <tr>
    <td>
     {assign var="greeting" value="{contact.email_greeting_display}"}{if $greeting}<p>{$greeting},</p>{/if}
-     <p>{ts}Below you will find a receipt for this contribution.{/ts}</p>
+      <p>
+        {if {contribution.contribution_page_id|boolean}}{contribution.contribution_page_id.receipt_text}
```

Comment:
This ignores new lines. In the online receipt we have {$receipt_text|htmlize}, would it make sense to define an equivalent for tokens?
I know ideally we'd set the html and text versions in the token itself, but that seems kind of roundabout here.

Reply:
Or maybe we take this opportunity to turn receipt_text into HTML in general and perhaps add a filter for text format output that strips the tags and adds newlines where required?

Reply:
`{contribution.contribution_page_id.receipt_text|boolean}`  might be more specific for the IF

Reply:
Also note - that if https://github.com/civicrm/civicrm-core/pull/27203 is accepted it should address your line issue